tests/inputs/test_importance.imcnp

```diff
@@ -41,7 +41,7 @@ MT3 lwtr.23t
 C execution
 ksrc 0 0 0
 imp:n,p 1 1 1 0 3
-imp:e   0 0 0 1 2
+imp:e   0 2r 1 r
```

Comment:
We also need to test a Jump. 

The following still is not recompressed correctly:

```
C Repeat recompressed incorrectly
VOL   J 1 2R 2 10 2R 20    3   
```

Leading to

```
Traceback (most recent call last):
  File "montepy/mcnp_object.py", line 50, in __init__
    self._tree = parser.parse(input.tokenize(), input)
  File "montepy/input_parser/parser_base.py", line 156, in parse
    tree = super().parse(token_generator)
  File "/opt/Mamba/forge3/envs/cafe/lib/python3.9/site-packages/sly/yacc.py", line 2137, in parse
    value = p.func(self, pslice)
  File "montepy/input_parser/parser_base.py", line 243, in shortcut_sequence
    short_cut = syntax_node.ShortcutNode(p)
  File "montepy/input_parser/syntax_node.py", line 1694, in __init__
    self._expand_repeat(p)
  File "montepy/input_parser/syntax_node.py", line 1758, in _expand_repeat
    raise ValueError(f"Repeat cannot follow a jump. Given: {list(p)}")
ValueError: Repeat cannot follow a jump. Given: [(shortcut:Shortcuts.JUMP: [(Value, None, padding: None)]), '3R']

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "mwe.py", line 6, in <module>
    deck = montepy.read_input("testo.i")
  File "montepy/input_parser/input_reader.py", line 31, in read_input
    problem.parse_input(replace=replace)
  File "montepy/mcnp_problem.py", line 293, in parse_input
    raise e
  File "montepy/mcnp_problem.py", line 278, in parse_input
    obj = obj_parser(input)
  File "montepy/data_inputs/data_parser.py", line 46, in parse_data
    return data_class(input)
  File "montepy/data_inputs/volume.py", line 33, in __init__
    super().__init__(input, in_cell_block, key, value)
  File "montepy/data_inputs/cell_modifier.py", line 32, in __init__
    super().__init__(input, fast_parse)
  File "montepy/data_inputs/data_input.py", line 58, in __init__
    super().__init__(input, self._parser)
  File "montepy/mcnp_object.py", line 53, in __init__
    raise MalformedInputError(
montepy.errors.MalformedInputError: 
: testo.i, line 33
Error parsing object of type: <class 'montepy.data_inputs.volume.Volume'>: Repeat cannot follow a jump. Given: [(shortcut:Shortcuts.JUMP: [(Value, None, padding: None)]), '3R']

the full input:
C Repeat recompressed incorrectly
VOL   J 3R 2 10 2R 20
```

Reply:
See: https://github.com/idaholab/MontePy/pull/520/files#diff-86f3bbc374cf733ce672ca79b57725426ef2a38df960fc9fe2e93406792ed300R920